### PR TITLE
[40] Calculate Product Shipping Cost

### PR DESCRIPTION
**Pull request by kazeemerinfolami**
## Task: `[40] Calculate Product Shipping Cost`
- `Calculate Product Shipping Cost`

### CODE REVIEW

File: submissions/kazeemErinfolami/javascript/Dom/index.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    }
+    else {
+        return false
+    }
+}
+
+function shippingPrice() {
+    let perPrice = priceValue * qty.value
+    if (perPrice < 1000) {
+        shippingRange = 100;
```
Comment (adaugochi) [inline submissions/kazeemErinfolami/javascript/Dom/index.js:41]: I can't see where `shippingRange` was declared. If it hasn't been declared before kindly use `let` to declare this variable name

File: submissions/kazeemErinfolami/javascript/Dom/index.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        shippingCost.innerHTML = shippingRange;
+    } else {
+        return false
+    }
+}
+
+function totalPrice() {
+    let perPrice = priceValue * qty.value
+    if (qty.value >= 1) {
+        cost.innerHTML = shippingRange + perPrice;
```
Comment (adaugochi) [inline submissions/kazeemErinfolami/javascript/Dom/index.js:60]: How come you are making use of `shippingRange` here. Where is the value coming from?

Comment (adaugochi) [reply submissions/kazeemErinfolami/javascript/Dom/index.js:60]: This function should just hold this `cost.innerHTML = shippingRange + perPrice;`. `shippingRange` and `perPrice` should be a parameter and the function should be called inside the `shippingPrice` function as `shippingRange` and `perPrice` is already calculated there

File: submissions/kazeemErinfolami/javascript/Dom/index.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+function calculate() {
+    productQuantity();
+    productPrice();
+    shippingPrice();
+    totalPrice();
+}
+
+qty.addEventListener("input", calculate);
+
```
Comment (adaugochi) [inline submissions/kazeemErinfolami/javascript/Dom/index.js:74]: remove extra white spaces below

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/kazeemErinfolami/javascript/Dom/index.js (hunk 2 above)
Comment (kazeemerinfolami) [reply submissions/kazeemErinfolami/javascript/Dom/index.js:60]: I started making use of the shippingRange in the "function shippingPrice()" block, I should have declared it as a variable, my bad.

**Review by adaugochi — APPROVED** [review]

**Pull request merged**